### DISCUSSION TITLE
Movimientos y estadísticas JS

### DISCUSSION THREAD

**Discussion by LuisFerVR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#454] [Q&A]
Disculpen, saben o podrían mostrarme en código cómo extraer de la pokeapi los movimientos por ejemplo que son muchos y son arreglos, como los extraigo a nuestra página para que salga en consola?

**Comment by doguedogue** [comment]
`let movimientos = data.moves.map(mov => mov.move.name);
Console.log(movimientos);
`

**Reply by LuisFerVR** [reply]
![image](https://user-images.githubusercontent.com/98798741/159096941-b46654dc-0df8-475c-bf57-386dbfda9d0f.png)

qué puede dar ese error ? estoy intentando con tus líneas de código, y  gracias.

**Reply by doguedogue** [reply]
Comenta poketip, veo que si obtienes la imagen.

**Reply by LuisFerVR** [reply]
gracias
